src/components/gui/gui.jsx

```diff
@@ -74,6 +76,8 @@ GUIComponent.propTypes = {
 };
 GUIComponent.defaultProps = {
     basePath: '/',
-    vm: new VM()
+    vm: new VM(),
+    storage: new Storage()
 };
+GUIComponent.defaultProps.vm.attachStorage(GUIComponent.defaultProps.storage);
```

Comment:
Ah, I see. That way you don't have a useless `defaultProps.storage` hanging around if you want a different one. Makes sense. I'd still like to officially* allow access to the storage module from outside the VM but thinking on it further I don't think this is the right place for it.

*Sure, you can do `vm.runtime.storage` but that doesn't exactly feel like a public API :)